### PR TITLE
Update practices and cleanup of the Performance section

### PR DESCRIPTION
**Pull request by josebolos**
- Change advice against using WOFF
- Remove Lighthouse from the list of recommended performance testing tools
- Add bold to the "concatenate files EVEN IF USING HTTP2" bit
- Add links backing up some of the performance-related claims
- Simplify language to help readability
- Make headings use sentence case consistently
- Improve unordered lists formatting
- Add missing TOCs
- Whitespace fixes

### CODE REVIEW

File: performance/font-loading.md [hunk 1]
```diff
@@ -42,9 +54,9 @@ We believe that FOIT should be avoided as it is the worst experience for the use
 
 We highly recommend reading [Zach Leat's comprehensive guide](https://www.zachleat.com/web/comprehensive-webfonts/) to get a good understanding of the different techniques available. There is also a good [follow-up article](https://css-tricks.com/the-best-font-loading-strategies-and-how-to-execute-them/) that summarises Zach's findings.
 
-### Serve WOFF and WOFF2 font formats
+### WOFF and WOFF2 font formats
 
-WOFF2 has the superior compression. However, unfortunately [it does not cover all of our Enhanced browsers](https://caniuse.com/woff2) at the time of writing. Therefore consider serving both formats. 
+WOFF2 has a superior compression, and [is supported by all of our Enhanced browsers](https://caniuse.com/woff2), so there's no need to serve fonts in WOFF format.
```
Comment (dotcode) [inline performance/font-loading.md:59]: What's the impact for our other browsers?

File: performance/images.md [hunk 2]
```diff
@@ -32,10 +32,9 @@ Most best practices described in this document apply to both categories unless o
 
 A number of more modern image formats can also be used to deliver images:
 
-* [WEBP](https://en.wikipedia.org/wiki/WebP) is a replacement for JPEG, PNG, and GIFs, but improvements to encoders like mozjpeg means that WEBP file sizes are only marginally smaller than JPEG. It's supported by almost all major browsers ([Safari supports it partially](https://caniuse.com/webp)) and can be used only as an enhancement to another better supported format.
+* [WebP](https://en.wikipedia.org/wiki/WebP) is a replacement for JPEG, PNG, and GIFs, but improvements to encoders like [mozjpeg](https://github.com/mozilla/mozjpeg) means that [WEBP file sizes are only marginally smaller than JPEG](https://siipo.la/blog/is-webp-really-better-than-jpeg). It's supported by almost all major browsers ([Safari supports it partially](https://caniuse.com/webp)) and can be used only as an enhancement to another better supported format.
```
Comment (dotcode) [inline performance/images.md:35]:
```suggestion
* [WebP](https://en.wikipedia.org/wiki/WebP) is a replacement for JPEG, PNG, and GIFs, but improvements to encoders like [mozjpeg](https://github.com/mozilla/mozjpeg) means that [WEBP file sizes are only marginally smaller than JPEG](https://siipo.la/blog/is-webp-really-better-than-jpeg). It's supported by almost all major browsers ([Safari supports it partially](https://caniuse.com/webp)) and must be used only as an enhancement to another better supported format.
```
Presume "must" was intended, due to the "only" qualifier on "can".

File: performance/images.md [hunk 3]
```diff
@@ -82,7 +81,7 @@ For content images, we encourage using an on-premise or cloud-based image server
 * Ensure that all images are optimised automatically.
 * Ensure that the appropriate cache headers are always set.
 
-Additionally, future improvements to these tools will result in improvements to the images being served automatically, without the images having to be manually re-encoded.
+As an additional benefit, future improvements to these tools may result in improvements to the images being served automatically, without the images having to be manually re-encoded.
```
Comment (dotcode) [inline performance/images.md:84]:
```suggestion
As an additional benefit, changes to these tools may result in improvements to the images being served automatically, without the images having to be manually re-encoded.
```
Just tweaking it to remove some redundancy and improve the way it reads.

Separately, is there any link that can explain why this statement is made?

**Review by dotcode — CHANGES_REQUESTED** [review]
Fantastic work, as ever, just some minor tweaks :)

File: performance/font-loading.md (hunk 1 above)
Comment (josebolos) [reply performance/font-loading.md:59]: There shouldn't be any impact as:

* The only browser to ever being served WOFF fonts was Internet Explorer.
* Older versions of IE are not able to access our sites at all, as they lack TLS 1.2 support.
* Later versions of IE are classed as "Core" browsers, and we don't serve webfonts to them.

**Comment by josebolos** [comment]
@dotcode I think I've addressed all your concerns. Would you mind giving this a thumbs up so it can be merged, please? TY!

**Review by dotcode — APPROVED** [review]

**Comment by dotcode** [comment]
Nice work, thanks Jose!

**Pull request merged**